.buildkite/scripts/dra.sh

```diff
@@ -13,8 +13,8 @@ set -eo pipefail
 # Either staging or snapshot
 TYPE="$1"
 
-## Read current version.
-VERSION=$(make get-version)
+## Read current version without the qualifier
+VERSION=$(make get-version-only)
```

Comment:
DRA requires a version with the format `major.minor.patch(-SNAPSHOT)` but without the `qualifier`. 

`make get-version` changed to support the qualifier, but it also changed the contract with the `DRA`.